lib/bind-it/jar_loader.rb

```diff
@@ -59,11 +59,11 @@ def self.set_java_logging
         file = Rjb::import('java.io.File')
       end
       ps = print_stream.new(file.new(self.log_file))
-      ps.write(::Time.now.strftime("[%m/%d/%Y at %I:%M%p]\n\n"))
-      system.setOut(ps)
+      # ps.write(::Time.now.strftime("[%m/%d/%Y at %I:%M%p]\n\n"))
```

Comment:
The reason it doesn't work is that JRuby 1.6 translates call of write(String) on java.io.PrintStream, which JRuby 1.7 doesn't.
I would suggest the following instead:
print_stream = Class.new(java.io.PrintStream) do
  java_alias(:write, :print, [java.lang.String])
end